src/java/org/apache/cassandra/cql3/conditions/ColumnCondition.java

```diff
@@ -495,6 +495,9 @@ private static boolean valueAppliesTo(CollectionType<?> type, Iterator<Cell<?>>
             if (value == null)
                 return !iter.hasNext();
 
+            if(operator.isContains() || operator.isContainsKey())
+                return valueAppliesTo(type, iter, value.get(ProtocolVersion.CURRENT), operator);
```

Comment:
Here you branch off to a logic specific to the CONTAINS operators, could we rename those methods to indicate that? just like the list/set/map methods i.e.